packages/alfa-dom/src/node/element.ts

```diff
@@ -370,7 +366,10 @@ function isSuggestedFocusableElement(element: Element): boolean {
       return element.attribute("href").isSome();
 
     case "input":
-      return element.attribute("type").every((attr) => attr.value !== "hidden");
+      return element
+        .attribute("type")
+        .flatMap((attribute) => attribute.enumerate("hidden"))
+        .isNone();
```

Comment:
Not strictly needed because the attribute value goes through `foldCase`. But that was a bit random coincidence that two transformations just happened to be the same…